internal/app/machined/pkg/system/runner/runner.go

```diff
@@ -56,6 +56,8 @@ type Options struct {
 	GracefulShutdownTimeout time.Duration
 	// Stdin is the process standard input.
 	Stdin io.ReadSeeker
+	// Specify an oom_score_adj for the container.
```

Comment:
```suggestion
	// Specify an oom_score_adj for the process.
```
We refer to everything as a "process" in this file. So, a consistency nit.

Reply:
Nice catch, thanks